blog/2023-01-10-first-offsite/index.mdx

```diff
@@ -0,0 +1,109 @@
+
+Fortunately, we had a small power bank and mobile hotspot with unlimited data. Unfortunately, our lead Rust engineer’s beefy Linux machine only gets a single hour of battery life on a full charge.
+
+When his machine died, we migrated to our local coffee shop, where power and WiFi were surprisingly abundant. _Editor’s note: Coffee shops in Santa Rosa are much more accommodating as a makeshift office than most in San Francisco._
+
+When you’re building software, electricity and internet are vital -- splurge for that bigger battery bank and faster hotspot!
+
+### Lumbar support is underrated
+
+You’ll sacrifice some ergonomics when you work out of an Airbnb. Put a premium on large work surfaces, open common areas, and comfortable chairs. After a few days of working hunched over on a sofa, we learned the hard way.
+
+Even better, leave heads down work until after the off-site. Instead, prioritize architecture discussions, brainstorming sessions, and design sprints. 
+
+### Stock the right fuel
+
+We got carried away at Costco. Our out-of-town guests had never witnessed the spectacle of an American snack aisle. Fortunately, our talented engineers can turn Funyuns into elegant lines of Rust code (shoutout [@Gabi](https://github.com/conectado)). 
+
+All jokes aside, you’ll need the right fuel for your hackathons and deep product discussions. We recommend stocking up with plenty of coffee and healthy snacks.
+
+### Plan ahead: it’ll pay off
+
+As is often the case with startups, we planned the trip on short notice. This meant many last-minute restaurant bookings and trips to the store for basic supplies.
+
+So, make a shopping list, book a few restaurants in advance, and have a backup plan. If you’re looking for a template, take a look at this [guide](https://posthog.com/handbook/company/offsites#how-to-plan-an-offsite-in-8-weeks---a-checklist) from our friends at PostHog. A great checklist is hard to beat.
+
+## What’s next?
+
+In a few short days, we:
+* Aligned on Firezone’s product direction
+* Made critical architecture decisions for the gateway and client apps
+* Made significant progress on a few key features
+* Bonded over great food, conversation, and Mario Kart
+
+We could have accomplished some of these over a few long Zoom calls and chat threads, but not as quickly. There’s something about mind melding over a few days of intense work that’s hard to replicate.
+
+So, will we have another off-site? Definitely!
+
+Until next time!
+
+:::info 👋 thanks for reading
+Firezone secures remote access to private resources. This post doesn't have much to do about security, networking, or a fancy new feature. But hey, our product improves the remote work experience - organizing a great off-site helps too.
+
+If you want to try it out, our [Docker install script](https://docs.firezone.dev/) gets Firezone running in just a few minutes.
+:::
```

Comment:
@jamilbk added a blurb to the bottom to frame the post + add some info on Firezone. Borrowed from fly.io.

I like how it gives us room to comment and connect things back to FIrezone for these "company building" type posts.

What do you think?

Reply:
Just needs the `?utm_source=blog`